inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -142,10 +142,10 @@ void FrontEnd::detectNetworkBatch(
                 VPU_THROW_FORMAT("Unsupported layer %s configuration: no outputs", layer->get_name());
 
             // 1. Don't support if DetectionOutput is not the last layer in network
-            for (const auto& outputHandle : layer->get_outputs()) {
-                for (const auto& inputHandle : outputHandle.get_inputs()) {
-                    auto outNode = inputHandle->get_node();
-                    if (std::dynamic_pointer_cast<::ngraph::opset3::Result>(outNode)) {
+            for (const auto& outputHandle : layer->outputs()) {
```

Comment:
`get_outputs()` was deprecated and I just replaced it to output(). 
You can open the current master branch and find that deprecation message recommends to use `outputs()` instead of `get_outputs()`